README.md

```diff
@@ -17,12 +17,20 @@ conda activate nf-schema-builder
 pre-commit install
 ```
 
+3. Build GUI assets:
+```bash
+cd gui
+npm install
+npm run build
```

Comment:
```suggestion
npm install --prefix gui
npm run build --prefix gui
```
or a `cd ..` in the end of the above, otherwise the other commands might fail